test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -324,6 +325,30 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 				Expect(processesFound).To(BeEmpty(), "The node %s has the following processes on the root cgroup: %v", node.Name, processesFound)
 			}
 		})
+
+		It("[test_id:74767] Should have the ocp-tuned-one-shot.service started and ended before kubelet", func() {
+			// https://issues.redhat.com/browse/OCPBUGS-26401
+			for _, node := range workerRTNodes {
+				By("verify that ocp-tuned-one-shot.service end time is before kubelet.service start time")
+				out, err := systemd.ShowProperty(context.TODO(), "ocp-tuned-one-shot.service", "ExecMainExitTimestampMonotonic", &node)
+				Expect(out).ToNot(BeEmpty())
+				Expect(err).ToNot(HaveOccurred())
+				tunedExitTimeProp := strings.Fields(out)[0]
+				tunedExitTimeStr := strings.Split(tunedExitTimeProp, "=")[1]
+				tunedExitTime, err := strconv.Atoi(tunedExitTimeStr)
+				Expect(err).ToNot(HaveOccurred())
+
+				out, err = systemd.ShowProperty(context.TODO(), "kubelet.service", "ExecMainStartTimestampMonotonic", &node)
+				Expect(out).ToNot(BeEmpty())
+				Expect(err).ToNot(HaveOccurred())
+				kubeletStartTimeProp := strings.Fields(out)[0]
+				kubeletStartTimeStr := strings.Split(kubeletStartTimeProp, "=")[1]
+				kubeletStartTime, err := strconv.Atoi(kubeletStartTimeStr)
+				Expect(err).ToNot(HaveOccurred())
+
+				Expect(tunedExitTime).To(BeNumerically("<", kubeletStartTime), "kubelet started before ocp-tuned-one-shot.service exits")
+			}
+		})
```

Comment:
thanks for the updates! I meant something like https://go.dev/play/p/TPxjcoK8m-w

Reply:
ah my bad I thought you meant that to replace the TrimSpace, will update that